styles.css

```diff
@@ -423,3 +427,305 @@ button a {
+  .intro-geometry {
+    display: block;
+    display: inline-flex;
+    max-width: 100%;
+    height: auto;
+  }
+
+  .about-section {
+    max-width: 100vw;
+    display: flex;
+    justify-content: space-evenly;
+    padding: 80px 0;
+    flex-direction: row-reverse;
+  }
+
+  .about-details {
+    max-width: 450px;
+    min-width: 300px;
+  }
+
+  .descriptive-hd {
+    font-size: 110px;
+    line-height: 100%;
+  }
+
+  .about-btn {
+    width: 50%;
+  }
+
+  .summary {
+    flex-direction: row;
+    justify-content: space-around;
+    width: 100%;
+    padding: 0;
+  }
+
+  .summary-cta {
+    max-width: 50%;
+  }
+
+  .summary-details {
+    flex: 1;
+  }
+
+  .name-box {
+    padding: 100px;
+    background-image: url('images/image_dots.svg');
+    background-position: bottom right;
+    background-repeat: no-repeat;
+  }
+
+  .summary-details .name {
+    font-size: 110px;
+    line-height: 100%;
+    padding: 0;
+    margin-bottom: 20px;
+  }
+
+  /* STYLING FOR PROJECT SECTION */
+  .head-image {
+    display: none;
+  }
+
+  .mobile-only {
+    display: none;
+  }
+
+  .flex-projects {
+    display: flex;
+    margin-top: 210px;
+  }
+
+  .flex-projects .cards-detail p {
+    font-weight: bold;
+    font-size: 37px;
+    line-height: 80%;
+    width: 90%;
+  }
+
+  .flex-projects .descriptive-hd {
+    transform: rotate(-90deg);
+  }
+
+  .flex-projects .cards-detail {
+    height: 353px;
+  }
+
+  .flex-projects .deco-bg {
+    background-image: url(images/image_dots.svg);
+    background-repeat: no-repeat;
+    background-position: bottom right;
+  }
+
+  .flex-projects1 {
+    height: 685px;
+    background: url('images/image-geometry_2.svg') top no-repeat, url('images/Rectangle22.png') left no-repeat, url('images/image-geometry_3.svg') bottom right no-repeat, url('images/Rectangle23.png') bottom left no-repeat;
+  }
+
+  .flex-projects1 .cards {
+    width: 420px;
+    float: right;
+  }
+
+  /* STYLING FOR TECHNOLGIES SECTION */
+  .tech {
+    display: grid;
+    margin-top: 0;
+    max-width: 60%;
+  }
+
+  .tech-details .flex {
+    padding: 0;
+    margin: 0;
+  }
+
+  .tech-details {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    gap: 0;
+    flex-wrap: wrap;
+  }
+
+  .tech-dimension {
+    width: 100%;
+  }
+
+  /* STYLING FOR FORM */
+  .contact-form {
+    display: flex;
+    background: url('images/image-geometry_6.svg') top left no-repeat, url('images/image-geometry_7.svg') bottom right no-repeat;
+    background-size: 200px;
+    align-items: center;
+    padding: 106px 14px 80px 18px;
+    margin: 50px 0 0 0;
+  }
+
+  .contact-form h2 {
+    font-size: 90px;
+    margin-bottom: 0;
+    margin-top: 20px;
+  }
+
+  .cta-summary {
+    width: 420px;
+    margin-top: 0;
+    margin-left: 16px;
+  }
+
+  .contact-form .input-wrapper {
+    background: #fff;
+    box-sizing: border-box;
+    font-size: 14px;
+    border: none;
+    width: 210px;
+    margin: 4px 0 4px 10px;
+  }
```

Comment:
Hi @HillaryOkello  

Thank you a lot for the insightful reviews.
I am and will do my best to ensure that I rectify the mentioned issues so that I can tap into my final submission review opportunity. 
>HappyCoding

Reply:
All done